src/backend/common/jit/BufferNodeBase.hpp

```diff
@@ -53,11 +53,11 @@ class BufferNodeBase : public common::Node {
         return m_linear_buffer && same_dims;
     }
 
-    void genKerName(std::stringstream &kerStream,
+    void genKerName(std::string &kerString,
                     const common::Node_ids &ids) const final {
-        kerStream << "_" << getNameStr();
-        kerStream << std::setw(3) << std::setfill('0') << std::dec << ids.id
-                  << std::dec;
+        kerString += '_';
```

Comment:
The filled in zeros are required to avoid naming conflicts. It is used to distinguish between 1, 3 and 13.